gammapy/maps/region/ndmap.py

```diff
@@ -516,6 +517,10 @@ def to_hdulist(self, format="gadf", hdu="SKYMAP", hdu_bands=None, hdu_region=Non
         if format in ["ogip", "ogip-sherpa", "ogip-arf", "ogip-arf-sherpa"]:
             hdulist.append(fits.BinTableHDU(table))
         elif format == "gadf":
+            if self.meta.setdefault("MJDREFI"):
```

Comment:
This does not seem right to me. This kind of meta data should not be in `RegionNDMap.meta`, but the stored in the `TimeMapAxis` instead.